stable/firehose/values.yaml

```diff
@@ -0,0 +1,60 @@
+replicaCount: 1
+namespace: firehose-test
+
+nameOverride: ""
+fullnameOverride: ""
```

Comment:
Can we add comments here for what they are used for?
That can also helm generating the docs using [helm-docs](https://github.com/norwoodj/helm-docs)